### DISCUSSION TITLE
Trouble using IPopt with Minizinc on Windows

### DISCUSSION THREAD

**Discussion by APeralta98** [coin-or/Ipopt#695] [Q&A]
Hello.
I am trying to use Ipopt as a non-linear Solver for Minizic, when trying to solve the model I get the following response.
![imagen](https://github.com/coin-or/Ipopt/assets/144490078/0330b0b0-5d47-4f5b-9cb3-3bd3f0fe493d)
I can't find any Minizinc Solver Configuration (.MSC) examples for IPopt apart from the generic example specified in the Minizinc documentation
![Sin título](https://github.com/coin-or/Ipopt/assets/144490078/97fa10c9-9af2-4ee2-b435-b9f377db675b)
I would appreciate the help of anyone who has experience using this solver in Minizinc or has any idea of ​​what to do.

**Comment by svigerske** [comment]
I would think that the one `ipopt.msc` would need to look similar to the example given for Couenne. `executable` would need to point to your Ipopt executable (`ipopt.exe`).

**Reply by APeralta98** [reply]
Yes i did that but i get the error message
![msc-used](https://github.com/coin-or/Ipopt/assets/144490078/1b1e76a8-1724-4ef9-a4a8-4d3227113375)

**Reply by svigerske** [reply]
Please check with the Minizinc support. This seems to be an issue with minizinc's solver configuration file and not with Ipopt (Ipopt isn't even starting).
